aspnetcore/security/key-vault-configuration.md

```diff
@@ -12,11 +12,13 @@ uid: security/key-vault-configuration
 
 :::moniker range=">= aspnetcore-6.0"
 
-This article explains how to use the [Azure Key Vault](/azure/key-vault/) configuration provider to load app configuration values from Azure Key Vault secrets. Azure Key Vault is a cloud-based service that helps safeguard cryptographic keys and secrets used by apps and services. Common scenarios for using Azure Key Vault with ASP.NET Core apps include:
+This article explains how to use the [Azure Key Vault](https://azure.microsoft.com/services/key-vault/) configuration provider to load app configuration values from Azure Key Vault secrets. Azure Key Vault is a cloud-based service that helps safeguard cryptographic keys and secrets used by apps and services. Common scenarios for using Azure Key Vault with ASP.NET Core apps include:
```

Comment:
Why replace the learn (documentation) page with the marketing page?

Reply:
The other section had the full URL. Not sure which is better. Given it's the first instance Key Vault is really mentioned, seemed like the landing page might be better. @jlichwa which way do you prefer?

Reply:
Original https://learn.microsoft.com/en-us/azure/key-vault/ (a learn landing page)
your change: https://azure.microsoft.com/en-us/products/key-vault/ : A marketing page

we avoid marketing pages like the plague, but I'll let you decide.

Reply:
> we avoid marketing pages like the plague, but I'll let you decide.

If @jlichwa doesn't have any objection as the product PM, I'll stick with that MO then.

Reply:
The learn page has two prominent and helpful links
<img width="242" alt="image" src="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/dd82c5bc-26e7-47d5-95bf-6aacfdda4ad5">